pom.xml

```diff
@@ -34,11 +34,10 @@
   <properties>
     <revision>1.17</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.150.1</jenkins.version>
+    <jenkins.version>2.222.4</jenkins.version>
     <java.level>8</java.level>
-    <workflow-step-api.version>2.19</workflow-step-api.version>
-    <workflow-cps.version>2.67</workflow-cps.version>
-    <workflow-support.version>3.3</workflow-support.version>
+    <pipeline-model-definition-version>1.3.8</pipeline-model-definition-version>
+    <jcasc.version>1.41</jcasc.version>
```

Comment:
as you are using the plugin BOM (which is great!), the JCasC version is normally defined in it. Is it not?

Reply:
Not sure, a bunch of stuff isn't

Reply:
it seems that you can. but it's not a blocker for me